octomap_server/src/octomap_saver.cpp

```diff
@@ -26,16 +26,16 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-#include <rclcpp/rclcpp.hpp>
-#include <octomap_msgs/conversions.h>
 #include <octomap/octomap.h>
 
-#include <octomap_msgs/srv/get_octomap.hpp>
-
 #include <chrono>
 #include <memory>
 #include <string>
 
+#include "rclcpp/rclcpp.hpp"
```

Comment:
I am surprised to see headers from other packages using " rather than <> -- is this a required change?

Reply:
I am not a nuanced C++ developer but my understanding is that `#include<>` is reserved for system headers and anything else "should" use `#include""` since the search paths for `#include<>` are compiler-dependent whereas `#include""` uses the paths specified in compiler options.

I've always seen rclcpp included with `#include""` https://github.com/ros2/examples/blob/14c743af8bbca6a40e83da7a470c5332dce66d9d/rclcpp/wait_set/src/talker.cpp#L17-L19 but that alone is just cargo-culting if my understanding above is incomplete.

Reply:
Thank you for the explanation - I am in a similar boat, and my present understanding was that quotation marks first searches for local files before going through the include paths specified by `-I`, whereas chevrons search the system/implementation-defined paths and then the `-I` defined include directories. There are also more detailed explanations on [SO](https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename)

Since there's no functional change happy to go with what is commonly used in ROS2 (vs in ROS1 I mostly saw chevrons for headers in other packages).